.clang-tidy

```diff
@@ -0,0 +1,27 @@
+---
+Checks:    
+      'modernize-*,
+       -modernize-use-equals-default,
+       -modernize-concat-nested-namespaces'
+      
+      # -modernize-use-equals-default        # auto-fix is broken (doesn't insert =default correctly)
+      # -modernize-concat-nested-namespaces  # auto-fix is broken (can delete code)
+
+WarningsAsErrors: ''
+HeaderFilterRegex: ''
+AnalyzeTemporaryDtors: false
+FormatStyle:     none
+CheckOptions:
+ - key:             modernize-loop-convert.MaxCopySize
+   value:           '16'
+ - key:             modernize-loop-convert.MinConfidence
+   value:           reasonable
+ - key:             modernize-loop-convert.NamingStyle
+   value:           CamelCase
```

Comment:
What do these do?

Reply:
See [the docs](https://clang.llvm.org/extra/clang-tidy/checks/modernize-loop-convert.html) for examples but it converts `push_backs` to loops and iterators to range based for loop. The `CamelCase` is for the style of the variable name created in the new loop. I just copied these settings from `rmm` to be as consistent as possible.

Reply:
That came from RMM? This rule  has never fired in RMM AFAIK. If it had converted names to CamelCase I would have noticed and changed that setting -- we don't use CamelCase in cuDF or RMM. Please change this.

Note the docs you linked don't mention the NamingStyle option.

Reply:
Here are more details on the NamingStyle: https://reviews.llvm.org/D13052

Reply:
Fixed.